src/main/java/fruitsassortment/ListOfFruits.java

```diff
@@ -0,0 +1,15 @@
+package fruitsassortment;
+
+public enum ListOfFruits {
+    BANANA,
+    APPLE;
+
+    public static boolean contains(String name) {
+        for (ListOfFruits fruitName : ListOfFruits.values()) {
+            if (fruitName.toString().equals(name)) {
+                return true;
+            }
+        }
+        return false;
+    }
```

Comment:
this might be called overengineering, we should assume that all the fruits passed are valid